hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -412,6 +412,34 @@ class ip_env_cfg extends cip_base_env_cfg #(.RAL_T(ip_reg_block));
     tl_intg_alert_fields[ral.fault_status.intg_err] = 1;
 ```
 
+### Security Verification for memory integrity
+The countermeasure of memory integrity stores the data integrity in the memory rather than generating the integrity on-the-fly during a read.
+The test [passthru_mem_intg_tests](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/dvsim/tests/passthru_mem_intg_tests.hjson) can fully verified this countermeasure.
+The details of the test sequences are described in the [tl_device_access_types_testplan](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/dvsim/testplans/passthru_mem_intg_testplan.hjson), users need to override the task `inject_intg_fault_in_passthru_mem` to inject an integrity to the memory in the block common_vseq.
+
+The following is an example from `sram_ctrl`, in which it flips up to `MAX_TL_ECC_ERRORS` bits of the data and backdoor write to the memory.
```

Comment:
Should this be something like: ".. of the data and generates a backdoor write to the memory" ? Otherwise I think this is missing a verb in the second clause.